src/QuickCheckVEngine/RVFI_DII/RVFI.hs

```diff
@@ -73,38 +85,33 @@ type RV_RegIdx = Word8
 -- | Type synonym for a RISCV XLEN-bit word
 type RV_WordXLEN = Word64
 
--- | Type synonym for a RISCV XLEN-bit word
-type RV_WordXLEN_ByteMask = Word8
+data RV_PrivMode = PRV_U | PRV_S | PRV_Reserved | PRV_M deriving (Enum, Show, Eq)
+
+type RV_XL = Word8
```

Comment:
I think those two types (`RV_PrivMode` and `RV_XL`, and associated helper functions `prvString` and `ixlString` I suppose) belong somewhere in `src/RISCV/`, most likely `src/RISCV/Helpers` (and can also most likely now lose the `RV_` prefix in favor of a `RISCV.` prefix instead)